lib/Timber.php

```diff
@@ -220,11 +220,44 @@ public static function get_term( $term, $taxonomy = 'post_tag', $TermClass = 'Ti
 	================================ */
 
 	/**
-	 * Get one or more users as a collection.
+	 * Gets one or more users as an array.
+	 *
+	 * By default, Timber will use the `Timber\User` class to create a your post objects. To
+	 * control which class is used for your post objects, use [Class Maps]().
```

Comment:
This is the one thing I was like, "oh, we should remember to make a guide/link for that" and then I saw you were one step ahead of me with yr `@todo`